lib/perl/Genome/Model/SomaticValidation/Command/IdentifyDnp.pm

```diff
@@ -55,7 +55,7 @@ sub params_for_result {
     my $build = $self->build;
     my @result_users = Genome::SoftwareResult::User->get(user => $build);
     my @results = map($_->software_result, @result_users);
-    my @merged_results = grep { $_->isa('Genome::InstrumentData::AlignedBamResult') } @results;
+    my @merged_results = grep { ! $_->isa('Genome::InstrumentData::AlignmentResult') } grep { $_->isa('Genome::InstrumentData::AlignedBamResult') } @results;
```

Comment:
Why can't this be looking for `Genome::InstrumentData::AlignmentResult::Merged`?  Filtering out select subclasses does not feel like the right solution.